Swift/IndexConfiguration.swift

```diff
@@ -36,19 +36,24 @@ public struct FullTextIndexConfiguration: IndexConfiguration, IndexConfigConvert
     /// Setting the language code affects how word breaks and word stems are parsed.
     /// Without setting the value, the current locale's language will be used. Setting
     /// a nil or "" value to disable the language features.
-    public var language: String?
+    public let language: String?
+    
+    /// A predicate expression defining conditions for indexing documents.
+    /// Only documents satisfying the predicate are included, enabling partial indexes.
+    public let `where`: String?
     
     /// Constructor for creating a full-text index by using an array of N1QL expression strings
-    public init(_ expressions: [String], ignoreAccents: Bool? = FullTextIndexConfiguration.defaultIgnoreAccents, language: String? = nil) {
+    public init(_ expressions: [String], ignoreAccents: Bool? = FullTextIndexConfiguration.defaultIgnoreAccents, language: String? = nil, where: String? = nil) {
```

Comment:
I'm not sure on this. If moving where next to expressions, will it break any users?

Reply:
I missed that only expressions can be NOT named, but the rest require it to. So no, it wouldnt make a difference. will move it